web_development_101/javascript_basics.md

```diff
@@ -32,6 +32,7 @@ This lesson will tend to focus on understanding the programming side of Javascri
 *Submit a link to the github repo with your files in it here with a pull request.  Please include your partner's github handle somewhere in the description if they would like attribution*
 
 * Add your solution below this line!
+My solution is here ! (Javier Reinoso's solution)
```

Comment:
@JavoFrola Can you please follow the format used in the other solutions? And, please add a link to your solution